spec/openapi.yaml

```diff
@@ -6468,6 +6468,84 @@ components:
             The subject field is used to denote subject of the MMS message and
             has a maximum size of 64 characters long
           example: New Products
+    DetailMessage:
+      title: Message
+      required:
```

Comment:
Given this is a model of message records (for responses) rather than request bodies, do we need the required attributes list? On the chance we do, wouldn't many more be present?

Reply:
I'm not too sure which ones are required here, it doesn't specify on the spreadsheet or on the original conversion doc

Reply:
I don't think we need to list required for responses because it is a machine generated data model.